src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py

```diff
@@ -57,6 +57,12 @@ def algorithm(self, parameters):
                 if response[0]['status'] == "ok" and len(response[0]['ids']) == len(jobIDs):
                     for docID in jobIDs:
                         self.fwjrAPI.updateArchiveUploadedStatus(docID)
+                    logging.info("...successfully uploaded %d docs", len(jobIDs))
+                    logging.debug("JobIDs uploaded:", jobIDs)
```

Comment:
I always mix the need or not of the format %? Don't we need them in these two debug lines? :-D